README.md

```diff
@@ -191,7 +191,7 @@ The above command pulled the existing data out of Route53 and placed the results
 | [EnvVarSource](/octodns/source/envvar.py) | | TXT | No | read-only environment variable injection |
 | [GoogleCloudProvider](/octodns/provider/googlecloud.py) | google-cloud-dns | A, AAAA, CAA, CNAME, MX, NAPTR, NS, PTR, SPF, SRV, TXT  | No | |
 | [MythicBeastsProvider](/octodns/provider/mythicbeasts.py) | Mythic Beasts | A, AAAA, ALIAS, CNAME, MX, NS, SRV, SSHFP, CAA, TXT | No | |
-| [Ns1Provider](/octodns/provider/ns1.py) | ns1-python | All | Yes | No CNAME support, missing `NA` geo target |
+| [Ns1Provider](/octodns/provider/ns1.py) | ns1-python | All | Yes | Missing `NA` geo target |
```

Comment:
Heh. I can't even remember at this point as we don't use dynamic CNAMEs. If you're used/tested `dynamic` CNAMEs with NS1 then lmk and we can ship this.

Reply:
haha yeah, same here, so I did "trust but verify" and used octodns to create, update, and delete a CNAME record on my NS1 account before I submitted this, without any issues